index.html

```diff
@@ -0,0 +1,1924 @@
+                <div class="opinion-info">
+                  <h5>
+                    On Memorial Day, Remember Why Americans Made the Ultimate
+                    Sacrifice
+                  </h5>
+                  <h4>BY NEWT GINGRICH</h4>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="spons2 container-fluid no-padding">
+            <!-- SPONSORED 2 -->
+            <article class="card card2">
+              <div class="img">
+                <a href="#"
+                  ><img
+                    class="card-img-bottom"
+                    src="images/spons11.jpg"
+                    alt="Card image"
+                    style="width: 100%;"
+                /></a>
+                <div class="cat cat2">SPONSORED INSIGHT</div>
+              </div>
+              <div class="card-body pad">
+                <h3 class="card-title">
+                  Looking for a Powerful, Lightweight and Long-lasting Cordless
+                  Vacuum?
+                </h3>
+                <p class="card-text">
+                  Roborock H6, 2020 Red Dot Award Winner, Ultra-Long lasting
+                  LiPo battery, available now on Amazon.
+                </p>
+              </div>
+            </article>
+            <form
+              class="subscribe form-inline d-flex flex-column justify-content-center"
+            >
+              <div
+                class="form-group d-flex align-items-center justify-content-between"
+              >
+                <img class="flex-shrink-0" src="images/news-icon.png" alt="" />
+                <h3 class="darkh3">Start your day with our top 5 articles</h3>
+              </div>
+              <div class="form-group">
+                <input
+                  type="email"
+                  class="form-control"
+                  id="inputPassword2"
+                  placeholder="Email Address"
+                />
+              </div>
+              <button type="submit" class="btn btn-primary">
+                FREE SIGN UP >
+              </button>
+            </form>
+            <article class="card card2">
+              <div class="img">
+                <a href="#"
+                  ><img
+                    class="card-img-bottom"
+                    src="images/spons12.jpg"
+                    alt="Card image"
+                    style="width: 100%;"
+                /></a>
+                <div class="cat cat2">SPONSORED INSIGHT</div>
+              </div>
+              <div class="card-body pad">
+                <h3 class="card-title">
+                  Looking for a Powerful, Lightweight and Long-lasting Cordless
+                  Vacuum?
+                </h3>
+                <p class="card-text">
+                  Roborock H6, 2020 Red Dot Award Winner, Ultra-Long lasting
+                  LiPo battery, available now on Amazon.
+                </p>
+              </div>
+            </article>
```

Comment:
The top of the image should stick to the top
![Annotation 2020-06-04 121601](https://user-images.githubusercontent.com/57726348/83724002-2fa91180-a65d-11ea-851d-a35d74380e8e.jpg)